src/services/categoriesService.js

```diff
@@ -0,0 +1,13 @@
+import { server } from './index';
+
+export const getCategoriesService = () => server.get('public/categories');
+
+export const getCategoryService = id => server.get(`public/categories/${id}`);
+
+export const getShopInfoService = () => server.get('shop_info');
+
+export const createCategoryService = (title = 'New category', products = []) => server.post('categories', { title, products });
```

Comment:
A category has also description field. And pass to service the whole object, do not pass `title`, `products` and other as separate arguments (as you did in `loginUserService`)